pkg/clustermesh/remote_cluster.go

```diff
@@ -129,7 +129,7 @@ func (rc *remoteCluster) releaseOldConnection() {
 	backend := rc.backend
 	rc.backend = nil
 
-	rc.metricTotalNodes.WithLabelValues(rc.mesh.conf.Name, rc.mesh.conf.NodeName, rc.name).Set(float64(rc.remoteNodes.NumEntries()))
```

Comment:
IF we are releasing the old connection, shouldn't we set it to zero? :thinking:

Reply:
My (wrong) assumption was that we might want to report the last known value and we would not see any updates afterwards. But you're right, setting it to zero is correct when we release the old connection. Though it probably makes sense to explicitly set 0 instead of relying on `(*store.SharedStore)(nil).NumEntries()` to return 0.